src/utils/cryptoModule.js

```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const lisk = require('lisk-js');
+import lisk from 'lisk-js';
```

Comment:
`import { crypto } from 'lisk-js'; `
or
`import { crypto as cryptoModule } from 'lisk-js';`
could make sense in this module

Reply:
Nice, I'll incorporate this in the decrypt passphrase PR as I've got a couple of refactors lined up there.